src/client/sandbox/node/index.js

```diff
@@ -2,6 +2,7 @@ import SandboxBase from '../base';
 import WindowSandbox from './window';
 import DocumentSandbox from './document';
 import ElementSandbox from './element';
+import NativeMethods from '../native-methods';
```

Comment:
We don't use upper case notation for object anymore, it's used only for the class contructors now (see other sources). So it should be `nativeMethods`